docs/apis/batch/libs/gelly.md

```diff
@@ -2067,7 +2067,92 @@ configuration.
 
   <tbody>
     <tr>
-      <td><strong>TranslateGraphIds</strong></td>
+      <td>degree.annotate.directed.<br/><strong>VertexInDegree</strong></td>
+      <td>
+        <p>Annotate vertices of a directed graph with the in-degree count.</p>
+{% highlight java %}
+DataSet<Vertex<K, LongValue>> inDegree = graph
+  .run(new VertexInDegree()
+    .setIncludeZeroDegreeVertices(true));
```

Comment:
What does the `setIncludeZeroDegreeVertices` do and what other options are available? Could you please document all of them (for the rest of algorithms also)?